Author: Daniel Schepler <dschepler@gmail.com>
Description: fix missing include causing build failure with eglibc 2.16
 eglibc 2.16 is more strict in its handling of <sys/resource.h>, so fix
 our includes here to address a build failure.
Bug-Debian: http://bugs.debian.org/693450

Index: pam.debian/modules/pam_unix/pam_unix_acct.c
===================================================================
--- pam.debian.orig/modules/pam_unix/pam_unix_acct.c
+++ pam.debian/modules/pam_unix/pam_unix_acct.c
@@ -47,6 +47,7 @@
 #include <time.h>		/* for time() */
 #include <errno.h>
 #include <sys/wait.h>
+#include <sys/resource.h>
 
 #include <security/_pam_macros.h>
 
Index: pam.debian/modules/pam_unix/pam_unix_passwd.c
===================================================================
--- pam.debian.orig/modules/pam_unix/pam_unix_passwd.c
+++ pam.debian/modules/pam_unix/pam_unix_passwd.c
@@ -54,6 +54,7 @@
 #include <ctype.h>
 #include <sys/time.h>
 #include <sys/stat.h>
+#include <sys/resource.h>
 
 #include <signal.h>
 #include <errno.h>
